packages/stdlib/_test.pony

```diff
@@ -63,6 +63,7 @@ actor Main is TestList
     itertools.Main.make().tests(test)
     json.Main.make().tests(test)
     logger.Main.make().tests(test)
+    math.Main.make().tests(test)
```

Comment:
d'oh! I thought this already existed. dur. I should have realized that it didn't based on the rest of the PR.

Sorry about that.

Reply:
No worries. Before now, math had no tests so it made sense that this line was not present.